apps/src/weblab/WebLabView.jsx

```diff
@@ -74,7 +77,25 @@ class WebLabView extends React.Component {
                 </div>
               }
               <div>
-                {!this.props.isReadOnlyWorkspace &&
+                {this.props.isFullScreenPreviewOn &&
+                  <PaneButton
```

Comment:
It might be nicer to define individual props by difference here, since some of them stay the same regardless of whether we're in fullscreen mode.

```jsx
<PaneButton
  iconClass={this.props.isFullScreenPreviewOn ? 'fa fa-compress' : 'fa fa-arrows-alt'}
  leftJustified={false}
  headerHasFocus={true}
  isRtl={false}
  onClick={this.props.isFullScreenPreviwOn ? this.props.onEndFullScreenPreview : this.props.onStartFullScreenPreview}
/>
```